scripts/update/Updater.php

```diff
@@ -623,6 +624,17 @@ public function update($initialVersion)
             OntologyUpdater::syncModels();
             $this->setVersion('5.2.0');
         }
+
+        if ($this->isVersion('5.2.0')) {
+            /** @var EventManager $eventManager */
+            $eventManager = $this->getServiceManager()->get(EventManager::SERVICE_ID);
+
+            // Attach event listener for SyncFailedEvent.
+            $eventManager->attach(SyncFailedEvent::class, [CentralSyncLogListener::SERVICE_ID, 'logSyncFailed']);
```

Comment:
I think you forgot to attach the `sendSyncFailedConfirmation` too.

Reply:
It is registered only on Client VM and has dedicated event listener - https://github.com/oat-sa/extension-tao-sync/blob/feature/TAO-7782/log_sync_failed_event_on_cloud_server/scripts/tool/SyncLog/RegisterSyncStatusListener.php#L38